pkgs/applications/misc/elogind/default.nix

```diff
@@ -48,12 +55,41 @@ stdenv.mkDerivation rec {
     libxslt.bin # xsltproc
     docbook5 docbook_xsl docbook_xsl_ns docbook_xml_dtd_42 docbook_xml_dtd_45 # needed for docbook without Internet
 
-    # fixes: man/meson.build:111:0: ERROR: Could not execute command "/build/source/tools/xml_helper.py".
     python3Packages.python
-    python3Packages.lxml
+    python3Packages.jinja2
   ];
 
-  buildInputs = [ libcap ] ++ (if enableSystemd then [ udev ] else [ eudev ]);
+  buildInputs = [ acl audit dbus libcap libselinux pam util-linux ]
+    ++ (if enableSystemd then [ udev ] else [ eudev ]);
+
+  postPatch = ''
+    substituteInPlace meson.build --replace-fail "install_emptydir(elogindstatedir)" ""
+  '';
+
+  patches = [
+    (fetchurl {
+      url = "https://github.com/chimera-linux/cports/raw/49d65fe38be815b9918a15ac2d2ff2b123fc559a/main/elogind/patches/strerror_r.patch";
+      hash = "sha256-amqXP12mLtrkWuAURb3/aoQeeTSRYlYqL2q2zrKbhxk=";
+    })
+    (fetchurl {
+      url = "https://github.com/chimera-linux/cports/raw/49d65fe38be815b9918a15ac2d2ff2b123fc559a/main/elogind/patches/strerror_r_1.patch";
+      hash = "sha256-tVUlmPValUPApqRX+Cqkzn7bkIILYSuCouvgRsdl9XE=";
+    })
```

Comment:
I don't understand the upstream-status trailer in those. musl specific patches are not upstream relevant for a program that support musl? That doesn't really make sense

Reply:
elogind is extracted from systemd, and these patches were originally written for systemd.  When they were adapted to elogind, the upstream-status trailer wasn't removed, even though it probably should have been given the patch is now being used for a different project.